spec/models/solidus_bolt/payment_source_spec.rb

```diff
@@ -1,30 +1,21 @@
 require 'spec_helper'
 
 RSpec.describe SolidusBolt::PaymentSource, type: :model do
-  let(:column_list) {
-    [
-      'id',
-      'payment_method_id',
-      'card_token',
-      'card_last4',
-      'card_bin',
-      'card_number',
-      'card_expiration',
-      'card_postal_code',
-      'card_id',
-      'create_bolt_account',
-      'created_at',
-      'updated_at'
-    ]
-  }
+  let(:payment_source) { build(:bolt_payment_source) }
 
-  describe 'Check Model Integrity' do
-    it 'has correct table name' do
-      expect(described_class.table_name).to eq 'solidus_bolt_payment_sources'
+  describe 'validations' do
+    context 'with payment_method_id present' do
+      it 'is valid' do
+        expect(payment_source.valid?).to be(true)
+      end
     end
 
-    it 'has correct attributes' do
-      expect(described_class.new.attributes.keys.sort).to eq column_list.sort
+    context 'with payment_method_id absent' do
+      before { payment_source.payment_method_id = nil }
+
+      it 'is invalid' do
+        expect(payment_source.valid?).to be(false)
```

Comment:
It doesn't check that the error is related to the absence of the payment_method_id.
You should also check the error, or add shoulda-matchers gem and use the provided matchers.

Reply:
fixed in https://github.com/nebulab/solidus_bolt/pull/89/commits/af71fface00b34627b0dffbf956540b9f70ad193